services/motion/builtin/move_attempt.go

```diff
@@ -17,6 +18,13 @@ type moveResponse struct {
 	success bool
 }
 
+func (mr moveResponse) String() string {
+	if mr.err == nil {
```

Comment:
No need for the conditional. This whole function can just be:

`return fmt.Sprintf("builtin.moveResponse{success: %t, err: %v}", mr.success, mr.err)`

https://go.dev/play/p/EIRtOgDxJ-M